### ISSUE TITLE
Empty locID's seem to disappear

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#1630]
We have recently had cases where LocID's were created for data entry, but went unused for at least a few weeks due to other priorities. When we went back, the LocID's were missing. 10903282, 10903278, and 10903272 for example were created but disappeared. Is it possible to trace what happened to them from backups?  Is there an internal process where they are removed if not used up in time?  If that is the case, would it be possible to get an email telling us they are going bye bye, and an option to extend their life?

Thanks,

Chris Conroy

**Comment by dustymc** [comment]
Yes, unused localities and events are auto-deleted, and duplicate IDs are auto-merged. There is some history and discussion at https://github.com/ArctosDB/arctos/issues/865, https://github.com/ArctosDB/arctos/issues/1017, and several linked issues.

Documentation is http://handbook.arctosdb.org/documentation/locality.html#maintenance

All internal IDs should be seen as inherently ephemeral (because they are!). I don't think it's possible to do what we're doing (automate maintenance and duplicate merger), what we want to do (build simpler locality editing forms), and also add stability to some of these particularly unstable IDs.

Adding a locality (or event) name will prevent all automation and protect the event/locality. Doing that should just be a click from the create form. If that doesn't solve this, can you describe your workflow?

**Comment by cjconroy** [comment]
OK, we can add a locality name to keep them from disappearing. But, just so
we know, how long does it take before they disappear? Is it a set time?  2
weeks?  Does this effect localities that have been used in data entry, but
the record is still in the bulk loader?

Chris

**Comment by dustymc** [comment]
I think it's currently set to check after 30 days. The planned forms will ultimately determine how aggressive cleanup needs to be - 24 hours is the number I used in preliminary testing. 

The bulkloader is currently not keyed into Localities; the cleanup scripts do not know it exists.
